src/App.vue

```diff
@@ -380,18 +421,20 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
     // reset errors in case this method is invoked more than once (ie, retry)
     this.resetFlags()
 
+    // Feature flag safety check
+    let supportedFilings = await getFeatureFlag('supported-filings')
+    if (!supportedFilings?.includes(this.$route.meta.filingType)) {
+      this.accountAuthorizationDialog = true
```

Comment:
This is not an account authorization problem.

See https://github.com/bcgov/business-edit-ui/blob/c17dd44ff39cb6062b0ec47925475fac74dadb8d/src/views/Alteration.vue#L268

Reply:
Fair enough, i used this one as it was generic enough without giving a user too much information. Not sure we want to explicitly state 'What' isn't available and 'why' though.

Reply:
That's fair. How about asking PO/UX?

Reply:
Good point, maybe we should ask. PO and designers. 
What 'do' we want to tell users trying to manually enter in routes that are not available yet 😆

Reply:
Suspicious characters out there..

Reply:
It would take a savvy character but our code IS public so ... yeah, you make a good point. 😁

Reply:
For the sake of consistency, i've followed the same pattern you went with in Alterations :)
Thanks for pointing this out!